tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/model/ESEmailAdressContactSearchTest.java

```diff
@@ -0,0 +1,28 @@
+package com.linagora.tmail.james.jmap.model;
+
+public class ESEmailAdressContactSearchTest implements EmailAddressContactSearchEngineContract{
+    @Override
+    public EmailAddressContactSearchEngine testee() {
+        return new ESEmailAddressContactSearchEngine();
+    }
+
+    @Override
+    public void indexShouldReturnMatched() {
+        EmailAddressContactSearchEngineContract.super.indexShouldReturnMatched();
```

Comment:
When you implement a contract you don't need to override all the tests in that contract, they will be played :)